cmd/fsi_test.go

```diff
@@ -520,6 +520,74 @@ fix these problems before saving this dataset
 	}
 }
 
+// Test that parse errors are also properly shown for schema.
+func TestStatusParseErrorForSchema(t *testing.T) {
+	if err := confirmQriNotRunning(); err != nil {
+		t.Skip(err.Error())
+	}
+
+	r := NewTestRepoRoot(t, "qri_test_status_parse_error_for_schema")
+	defer r.Delete()
+
+	ctx, done := context.WithCancel(context.Background())
+	defer done()
+
+	pwd, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	rootPath, err := ioutil.TempDir("", "")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Save a dataset containing a body.json and meta component
+	cmdR := r.CreateCommandRunner(ctx)
+	err = executeCommand(cmdR, "qri save --body=testdata/movies/body_ten.csv me/ten_movies")
+	if err != nil {
+		t.Fatalf(err.Error())
+	}
+
+	// Change to a temporary directory.
+	os.Chdir(rootPath)
```

Comment:
I'm foreseeing us doing a _lot_ of "isolated directory tests" for FSI integration checks. we should think about adding something to `TestRepoRoot` that includes a "dataset directory" & methods to change to / from it.

I think this is a great start, this suggestion would make a nice follow-up PR